jetty-quickstart/src/main/java/org/eclipse/jetty/quickstart/QuickStartGeneratorConfiguration.java

```diff
@@ -154,9 +154,13 @@ public void generateQuickStartWebXml(WebAppContext context, OutputStream stream)
         Map<String, String> webappAttr = new HashMap<>();
         int major = context.getServletContext().getEffectiveMajorVersion();
         int minor = context.getServletContext().getEffectiveMinorVersion();
-        webappAttr.put("xmlns", "http://xmlns.jcp.org/xml/ns/javaee");
+        String ns = "https://jakarta.ee/xml/ns/jakartaee";
+        if (major < 5)
+            ns = "http://xmlns.jcp.org/xml/ns/javaee";
```

Comment:
This is faster and clearer
```suggestion
        String ns = (major < 5) ? "http://xmlns.jcp.org/xml/ns/javaee" : "https://jakarta.ee/xml/ns/jakartaee";
```